tests/providers/pagerduty/notifications/test_pagerduty.py

```diff
@@ -0,0 +1,134 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+from unittest import mock
+
+import pytest
+
+from airflow.models import Connection
+from airflow.operators.empty import EmptyOperator
+from airflow.providers.pagerduty.hooks.pagerduty_events import PagerdutyEventsHook
+from airflow.providers.pagerduty.notifications.pagerduty import (
+    PagerdutyNotifier,
+    send_pagerduty_notification,
+)
+
+PAGERDUTY_API_DEFAULT_CONN_ID = PagerdutyEventsHook.default_conn_name
+
+
+@pytest.fixture(scope="module", autouse=True)
+def pagerduty_api_connections():
```

Comment:
I doubt you need this fixture. See #31268 which I just opened.

Reply:
Yes, we don't removed it.